lib/elixir/lib/enum.ex

```diff
@@ -1890,9 +1888,19 @@ defmodule Enum do
       iex> Enum.slice(1..10, 5..20)
       [6, 7, 8, 9, 10]
 
+      # last five elements (negative positions)
+      iex> Enum.slice(1..30, -5..-1)
+      [26, 27, 28, 29, 30]
+
+      # last five elements (mixed positive and negative positions)
+      iex> Enum.slice(1..30, 25..-1)
+      [26, 27, 28, 29, 30]
+
+      # out of bounds
       iex> Enum.slice(1..10, 11..20)
       []
 
+      # range.first is greater than range.last
```

Comment:
We definitely use comments elsewhere, but I'm not sure we use them with the `iex>` format because the comment is then "outside" of IEx in the example. Maybe we can put the comment inline, like this:

```elixir
iex> foo # bar
result
```

wdyt?